src/routes/index.ts

```diff
@@ -339,7 +339,7 @@ router.route(config.UPDATE_OVERSEAS_ENTITY_CONFIRM_URL)
     navigation.hasOverseasEntity
   )
   .get(confirmOverseasEntityDetails.get)
-  .post(companyAuthentication, confirmOverseasEntityDetails.post);
```

Comment:
Why remove companyAuthentication?

Reply:
Because `companyAuthentication` is hard-coded to re-direct to `UPDATE_FILING_DATE_URL`, so the POST action of `confirmOverseasEntityDetails` was never being hit.

Removed the middleware check from the POST here and has no impact on the front-end as the next page's GET has the same middleware check.